src/DataStructures/ComplexDataVector.hpp

```diff
@@ -63,28 +63,6 @@ class ComplexDataVector
 namespace blaze {
 VECTOR_BLAZE_TRAIT_SPECIALIZE_ARITHMETIC_TRAITS(ComplexDataVector);
 VECTOR_BLAZE_TRAIT_SPECIALIZE_ALL_MAP_TRAITS(ComplexDataVector);
-#if ((BLAZE_MAJOR_VERSION == 3) && (BLAZE_MINOR_VERSION <= 3))
```

Comment:
I haven't had a chance to look through this in detail, but can you bump the required version in `cmake/SetupBlaze.cmake` and in our installation instructions?